OpenRA.Mods.Common/Activities/FindAndDeliverResources.cs

```diff
@@ -214,7 +214,7 @@ public override bool Tick(Actor self)
 
 						if (b != WVec.Zero && c != WVec.Zero)
 						{
-							var cosA = (int)(1024 * (b.LengthSquared + c.LengthSquared - a.LengthSquared) / (2 * b.Length * c.Length));
+							var cosA = (int)(1024 * (b.LengthSquared + c.LengthSquared - a.LengthSquared) / 2 / b.Length / c.Length);
```

Comment:
This can simplify to `(int)(512 * (b.LengthSquared + c.LengthSquared - a.LengthSquared) / b.Length / c.Length);`

Reply:
Ok, I thought that was intentional. Updated again.